apps/sv/src/main/scala/org/lfdecentralizedtrust/splice/sv/store/db/DbSvDsoStore.scala

```diff
@@ -520,7 +520,16 @@ class DbSvDsoStore(
     val opName = s"list${templateId.getEntityName}GroupedByRound"
     val partyFilter =
       if (ignoredParties.nonEmpty)
-        (sql"and reward_party not in " ++ inClause(ignoredParties)).toActionBuilder
+        companion match {
+          case SvRewardCoupon.COMPANION =>
+            // only SV reward coupons have a beneficiary field
+            (sql"and reward_party not in " ++ inClause(ignoredParties) ++
```

Comment:
should this be a and? wouldn't the reward party be the sv so we get back to having to ignore the sv? :thinking:

Reply:
RIght, not and not, makes sense